cmd/ocm/list/cmd.go

```diff
@@ -46,4 +47,7 @@ func init() {
 	Cmd.AddCommand(upgradepolicy.Cmd)
 	Cmd.AddCommand(user.Cmd)
 	Cmd.AddCommand(version.Cmd)
+	rhRegionCmd := rhRegion.Cmd
+	rhRegionCmd.Hidden = true
```

Comment:
should this be set by default in the cmd?  seems odd to toggle off here.

Reply:
We don't want the command `ocm list rh-regions` to be visible. We can hide the flags but this is the only way to hide the subcommands : https://github.com/spf13/cobra/issues/1507#issuecomment-1632443187

Reply:
if this is always hidden by default, i'm suggesting you default to hidden/true in the cmd itself.  it seems weird to toggle at this level.

Reply:
Ack. Added hidden in Cmd itself